README.md

```diff
@@ -533,6 +533,16 @@ Product | Supported Version | License
 
 ---
 
+## Governance
+
+The oneMKL Interfaces project is governed by the UXL Foundation and you can get involved in this project in multiple ways. It is possible to join the [Math Special Interest Group (SIG)](https://github.com/uxlfoundation/foundation/tree/main/math) meetings where the groups discuss and demonstrate work using this project. Members can also join the Open Source and Specification Working Group meetings.
+
+You can also join the mailing lists for the [UXL Foundation](https://lists.uxlfoundation.org/g/main/subgroups) to be informed of when meetings are happening and receive the latest information and discussions.
+
+You can contribute to this project and also contribute to the specification for this project, read the [CONTRIBUTING](CONTRIBUTING.md) page for more information.
```

Comment:
With this sentence, we could consider removing the Contributing section below.

Reply:
I left the contributing as is for now since we plan to add more links to maintainers, how to contact them, introduce slack channel, etc.